src/unstract/sdk/adapters/x2text/llm_whisperer/src/static/json_schema.json

```diff
@@ -18,7 +18,7 @@
       "title": "URL",
       "format": "uri",
       "default": "https://llmwhisperer-api.unstract.com",
```

Comment:
NIT: @harini-venkataraman  default url will be the bridge url. 
cc: @jaseemjaskp